src/main/scala/com/fulcrumgenomics/sopt/util/MarkDownProcessor.scala

```diff
@@ -59,6 +59,9 @@ class MarkDownProcessor(lineLength: Int = 80, indentSize: Int = 2) {
   private val EmptyChunk = Seq(Chunk.empty)
   private val SpaceChunk = Seq(Chunk.unwrappable(0, " "))
 
+  /** Trims trailing whitespace, but keeps the last ASCII escape code. */
+  private val trimRegex = raw"\s+(\u001B\[\d+m)*$$".r
```

Comment:
Shouldn't your `\s` be `\\s`?  Also possibly `\\[` and `\\d`?

Reply:
Could you add a little more documentation please.  I'm not entirely sure what this is doing?  Some examples would be nice, e.g.:

```scala
"hello world   " => "hello world"
"hello world  <esc-code>   " => ???
"hello world  <esc-code>  <esc-code>   " => ???
```